ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__odo-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make bin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
+images:
+- context_dir: openshift-ci/build-root/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: oc-bin-image
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 800m
+      memory: 6Gi
+tests:
+- as: odo-lp-interop-aws
```

Comment:
@smatula I would suggest modifying this test name to reduce duplication. Right now the periodic job name will be: `periodic-ci-redhat-developer-odo-main-odo-ocp4.13-lp-interop-odo-lp-interop-aws`. I would suggest renaming this test to be just `odo` or `odo-interop-aws`. We should probably double check what the standard format the team is following by for all jobs.

Doing this would end up with a job name like: `periodic-ci-redhat-developer-odo-main-odo-ocp4.13-lp-interop-odo`